python/lsst/atmospec/utils.py

```diff
@@ -546,10 +554,10 @@ def getLinearStagePosition(exp):
         The position of the linear stage, in mm.
     """
     md = exp.getMetadata()
-    linearStagePosition = 115  # this seems to be the rough zero-point for some reason
+    linearStagePosition = 116.1  # this seems to be the rough zero-point for some reason
     if 'LINSPOS' in md:
         position = md['LINSPOS']  # linear stage position in mm from CCD, larger->further from CCD
-        if position is not None:
+        if position is not None and position > 0:  # if negative, there is an issue with the linear stage:
```

Comment:
Do you really want to silently ignore negative positions? I'd either add a warning, or even raise if you think that's going to be a problem for the processing.

Reply:
If the value is negative, it means that the filter wheel is blocked at the `linearStagePosition = 116.1` because of some hardware bug (it was seen in 2025/01 data)

Reply:
This feels like something that was true at some point in time, but isn't necessarily always going to be true, so feels quite dodgy, but if that's how you'd like it for now, that's OK.